src/renderer/src/App.tsx

```diff
@@ -1,12 +1,52 @@
 import Versions from './components/Versions'
+import { IntlConfig, IntlProvider } from 'react-intl'
+import { createContext, Dispatch, SetStateAction, useEffect, useMemo, useState } from 'react'
 
-function App(): JSX.Element {
-  return (
-    <div className="container">
-      <Versions></Versions>
-      {/* Render other stuff here! */}
-    </div>
-  )
+import * as es from '../translations/es.json'
+import * as en from '../translations/en.json'
```

Comment:
I don't how the bundler works, but can we dynamically import these? Otherwise the user is putting all the translations in memory even if they are not using them.

Reply:
Also, is this format readable by the `intlProvider`? And if so, then should we just take out the `build:translations` script/look into why they suggest we use that script